cmd/manager/main.go

```diff
@@ -98,6 +99,13 @@ func main() {
 	go registrySyncer.Sync(stopCh)
 	go catalogsourceconfig.Syncer.Sync(stopCh)
 
+	if os.Getenv("RELEASE_VERSION") != "" {
```

Comment:
Why is this check required?

Reply:
@aravindhp This is to make sure this code runs only on an OKD cluster. I noticed we set this env variable for OKD clusters and not when the operator is installed in a plain kube cluster. 
Sort of a hacky way, but the code assumes a lot of the resources are in the `openshift-marketplace` namespace. 
Although, based on your comment above about making a best effort to delete resources instead of erroring out and exiting, we could get rid of this check. The log will just look messy in a plain kube cluster.

Reply:
We should do the same steps we are performing on an OKD cluster on a plain kuber cluster. We shouldn't be making assumptions on the namespace where the operator is running. We don't do this in other parts of the code. Not sure why you think you are making that assumption here.

Reply:
Why can't we just use a different namespace if we find it's a non openshift cluster?

Reply:
I don't want to make any assumptions on the name of the namespace.